src/Symfony/Component/Console/Tests/Helper/TableTest.php

```diff
@@ -550,6 +550,9 @@ public function testRowSeparator()
 TABLE;
 
         $this->assertEquals($expected, $this->getOutputContent($output));
+
+        // Test if fluent interface on addRow() with a single TableSeparator() works ok
```

Comment:
Yes, IMO you should add a message, something like this: "->addRow() with a TableSeparator returns the Table instance."